playbook/app/pb_kits/playbook/pb_date_picker/_date_picker.tsx

```diff
@@ -5,13 +5,13 @@ import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
 import { deprecatedProps, globalProps, GlobalProps } from '../utilities/globalProps'
 
 import datePickerHelper from './date_picker_helper'
-
 import Icon from '../pb_icon/_icon'
-import TextInput from '../pb_text_input/_text_input'
+import Caption from '../pb_caption/_caption'
+import Body from '../pb_body/_body'
```

Comment:
Let's go ahead and combine all these and reference our index.js with our named exports. 

```
import { Body, Caption, Icon } from '../../'
```